src/implementations/twig/pages/page-inner/page-inner.test.js

```diff
@@ -0,0 +1,22 @@
+import { renderTwigFileAsNode, renderTwigFileAsHtml } from '@ecl/test-utils';
```

Comment:
it takes fifteen second to run this test in my env, i wonder a bit what we are really testing here, we test components, i do not see the value in doing the same for a "page example".